src/main/java/school/lemon/changerequest/java/introduction/pr1/HammingDistance.java

```diff
@@ -1,7 +1,31 @@
 package school.lemon.changerequest.java.introduction.pr1;
 
+import java.util.Scanner;
+
 public class HammingDistance {
     public static int compute(String a, String a1) {
-        return -1;
+        //System.out.println("Input first word");
+        //Scanner scanner = new Scanner(System.in);
+        //String a = scanner.nextLine();
+
+        //System.out.println("Input second word");
+        //Scanner scanner2 = new Scanner(System.in);
+        //String a1 = scanner2.nextLine();
+
+        int lena=a.length();
+        int lena1=a1.length();
```

Comment:
Variables naming in Java should follow Java Code Convention - use camelStyle in variables naming.
So, here, they should be named as `lenA` and `lenA1`.